Application/Business/GameUserServices/GameUserServices.cs

```diff
@@ -28,8 +28,13 @@ public async Task<ICollection<GameUserEntity>> List()
     }
     public async Task<List<UserEntity>> ListUsersByGameId(Guid id)
     {
-        var game = await _gameUserRepo.WhereAsync(gu => gu.GameId == id,new string[] {"User"});
-        return game.Select(g => g.User).ToList();
+        var user = await _gameUserRepo.WhereAsync(gu => gu.GameId == id,new string[] {"User"});
+        return user.Select(g => g.User).ToList();
+    }
+    public async Task<List<GameEntity>> ListGamesByUserId(Guid id)
```

Comment:
Do not add a method not to be used anywhere

Reply:
AnhDP ask me to do it for her this morning though. And this method is use in this API in UserController
   ```
 [HttpGet("{id}/games")]
    public async Task<IActionResult> GetGameByUserID(Guid id)
    {
        return Ok(await _gameUserServices.ListGamesByUserId(id));
    }
```